.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -16,14 +16,21 @@ AGENTS_LABELS = [
     "ubuntu-nonsgx":    env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX",
     "windows-nonsgx":   env.WINDOWS_NONSGX_CUSTOM_LABEL ?: "nonSGX-Windows"
 ]
+VANILLA_AGENTS_LABELS = [
```

Comment:
I am wondering if it is best to have two different structures VANILLA and the other, as opposed to a single lookup map.

Reply:
An option would be to have it in the same map, but just use `acc-rhel-8-vanilla`?

Reply:
Yes, this is what I was thinking.